pkg/deploy/graph/analysis/dc.go

```diff
@@ -35,6 +38,24 @@ dc:
 				// The image stream for the tag of interest does not exist.
 				// TODO: Suggest `oc create imagestream` once we have that.
 				if isNode, exists := doesImageStreamExist(g, uncastIstNode); !exists {
+
+					// We may not be able to see the image stream because it's in different project.
+					for _, t := range dcNode.DeploymentConfig.Spec.Triggers {
+						if t.Type == deployapi.DeploymentTriggerOnImageChange &&
+							t.ImageChangeParams.From.Namespace != dcNode.DeploymentConfig.Namespace {
+							markers = append(markers, osgraph.Marker{
+								Node:         uncastDcNode,
+								RelatedNodes: []graph.Node{uncastIstNode, isNode},
+
+								Severity:   osgraph.WarningSeverity,
+								Key:        DifferentProjectImageWarning,
+								Message:    fmt.Sprintf("%s expects an image from a different project.", f.ResourceName(dcNode)),
+								Suggestion: osgraph.Suggestion(fmt.Sprintf("oc status --all-namespaces")),
```

Comment:
This will almost certainly fail.  Only cluster admins will be able to perform this action.